pushy/src/main/java/com/relayrides/pushy/apns/ApnsPushNotification.java

```diff
@@ -120,4 +120,23 @@
      * @since 0.5
      */
     String getTopic();
+
+    /**
+     * <p>
+     * Returns the id of the notification. This is generally used for notification management
```

Comment:
thanks, couldn't find actual written something in the docs regarding the apns-collapse-id